### PR TITLE
update QuickBrick docs from repo

### PR DESCRIPTION
**Pull request by neerfri**
## Description
Updating latest documentation from the [QuickBrick github repo](https://github.com/applicaster/QuickBrick/tree/b2ddf44/DOCS)
## Checklist

* [ ] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation is included

* [ ] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

**Comment by snooky23** [comment]
@neerfri There are lots of changes did someone test it?

File: book/SUMMARY.md [hunk 1]
```diff
@@ -180,37 +213,3 @@
...
-      - [Plugin creation guide](quick-brick/tvOS/Plugins/Video/VideoPluginsHowCreate.md)
-      - [Default Player](quick-brick/tvOS/Plugins/Video/DefaultPlayer/DefaultPlayer.md)
-      - [Player Dependent](quick-brick/tvOS/Plugins/PlayerDependents/intro.md)
-        - [Plugin creation guide](quick-brick/tvOS/Plugins/PlayerDependents/DependentPluginsHowCreate.md)
-        - [GoogleIMA](quick-brick/tvOS/Plugins/PlayerDependents/VideoAds/GoogleIMA/GoogleIMA.md)
-  - [Environment](/quick-brick/tvOS/Environment/intro.md)
-  - [Apple TV Debug](/quick-brick/tvOS/Environment/DebugBuildIntegration.md)
-  - [Apple TV Release](quick-brick/tvOS/Environment/ReleaseBuildIntegration.md)
-- [Beta Guidelines & information](/quick-brick/Beta-guidelines.md)
-- [Setup your project](/quick-brick/plugin-setup.md)
```
Comment (snooky23) [inline book/SUMMARY.md:204]: This is the latest plugin guide I have written, it's updated and should stay.

Comment (neerfri) [reply book/SUMMARY.md:204]: It stays. It is also linked to from `Getting Started with Zapp Plugins` > `Get Ready to Work` > `Quick-Brick` so gitbook showed that one selected when I clicked the menu item under QB.

I wasn't sure if it's better to keep the one in the `Getting Started with Zapp Plugins` section or the one in the QB section

**Comment by neerfri** [comment]
@snooky23 who is this document intended for? customers or applicaster employees?
https://developer.applicaster.com/getting-started/set-up-bintray-and-circle-ci.html

File: book/SUMMARY.md [hunk 2]
```diff
@@ -180,37 +208,3 @@
...
-  - [Run Quick-Brick Natively](/quick-brick/native-project-dev.md)
```
Comment (snooky23) [inline book/SUMMARY.md:205]: `Run Quick-Brick Natively` should stay as well. please check where it links.

File: book/SUMMARY.md (hunk 1 above)
Comment (snooky23) [reply book/SUMMARY.md:204]: This is a very basic doc to get started with the plugins setup. 
You can add extra information inside the description chapter about where we manage the q.b qiki.
For example: for more information check our QuickBrick docs. (add a link to the main quick-brick page on the left nav-bar)

Comment (snooky23) [reply book/SUMMARY.md:204]: Please check that the docs assets are still available

File: book/SUMMARY.md (hunk 2 above)
Comment (snooky23) [reply book/SUMMARY.md:205]: Please check that the docs assets are still available

Comment (neerfri) [reply book/SUMMARY.md:205]: > `Run Quick-Brick Natively` should stay as well. please check where it links.

Done

File: book/SUMMARY.md [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+
+## Zapp Plugins Getting Started
+
 - [Getting Started with Zapp Plugins](/getting-started/zapp-plugins.md)
   - [Plugin types](/getting-started/plugin-types.md)
   - [Plugin abilities](/getting-started/plugin-abilities.md)
   - [Get Ready to Work](/dev-env/intro.md)
     - [iOS](/dev-env/iOS.md)
-      - [Applicaster Zapp App Connector utilities for plugins](/dev-env/iOS/utils.md)
+    - [Applicaster Zapp App Connector utilities for plugins](/dev-env/iOS/utils.md)
```
Comment (snooky23) [inline book/SUMMARY.md]: This should be below iOS since it's part of the iOS logic.

**Pull request merged**